common_analysis_ip_and_uri_finder/ip_and_uri_finder_analysis.py

```diff
@@ -89,23 +81,21 @@ def __init__(self, yara_uri_rules):
     def find_uris(self, uri_string):
         try:
             yara_matches = self.rules.match(data=uri_string)
-        except Exception as e:
-            logging.error("Could not match yara rules: {} {}".format(exc_info()[0].__name__, e))
+        except Exception:
+            logging.error('Could not match yara rules', exc_info=True)
```

Comment:
This does the same thing as
```suggestion
            logging.exception('Could not match yara rules')
```